src/ralph/operations/changemanagement/subscribtions.py

```diff
@@ -0,0 +1,102 @@
+from importlib import import_module
+
+import pyhermes
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from ralph.assets.models import BaseObject
+from ralph.operations.models import Operation, OperationType
+
+
+def _safe_load_user(username):
+    """Loads an existing user or creates a new one."""
+
+    if username is None:
+        return
+
+    model = get_user_model()
+
+    user, created = model.objects.get_or_create(username=username)
+
+    if created:
+        user.is_active = False
```

Comment:
couldn't you pass it to `get_or_create`? Sth like `user, created = model.objects.get_or_create(username=username, defaults={'is_active': False})`